airbyte-integrations/connectors/source-http-request/source_http_request/source.py

```diff
@@ -134,6 +134,11 @@ def get_json_schema(self):
                         my_dict = [root[i][0] for i in sorted(root.keys())][0]
                         df = pd.DataFrame.from_dict([my_dict])
                 headers = df.columns.tolist()
+            elif self._response_format == "xlsx":
+                data_xls = pd.read_excel(resp.content,  dtype=str, index_col=None)
+                data_xls.to_csv('temp.csv', encoding='utf-8', index=False)
+                df = pd.read_csv('./temp.csv', dtype=str, index_col=None)
+                headers = df.columns.tolist()
```

Comment:
Why do we transform a dataframe to csv and then this csv to dataframe again?

Reply:
Since we are using pandas, to make a xlsx readable we need to make it a .csv

The method to_csv() writes a csv file that's why we need to read it again.